doc/liquid/drops.md

```diff
@@ -362,6 +362,9 @@ Returns the url of the API spec.
 ### system_name
 Returns the name of the spec.
 
+### service
+Returns the service of the spec if it has any or nil otherwise.
```

Comment:
Can we backquote `nil`?

Reply:
No idea 😱 This is autogenerated, but I will try to accomplish that 😄 Maybe it is as easy as typing backquoted the original text 😊 I will try 😄

Reply:
You were right, it was that easy and it looks better 👍 💪